sshagentmux/util.py

```diff
@@ -52,7 +52,7 @@ def daemonize(target=None, pidfile=None, stdin='/dev/null', stdout='/dev/null',
         raise RuntimeError('fork #1 failed.')
 
     os.chdir('/')
-    os.umask(077)
+    os.umask(0o77)
```

Comment:
is that a lower case `o` (the alpha). ?  Correct?

Reply:
That's an octal literal. Looks like it will accept either case, but lower case is more clear than `0O77` https://peps.python.org/pep-3127/